### PR TITLE
responsive design done

### PR DESCRIPTION
**Pull request by jersonlopez**
## Summary
- responsive design added to book detail using sass variables

## Screenshot
image

## Trello Card
https://trello.com/c/zMIYJjpr/34-responsive

### CODE REVIEW

File: maquetado/scss/detail.scss [hunk 1]
```diff
@@ -93,3 +95,74 @@
   font-size: 20px;
   line-height: 23px;
 }
+
+@media only screen and (max-width: 1024px) {
+  .body {
```
Comment (CrisVega08) [inline maquetado/scss/detail.scss]: Try to avoid styles on tags

File: maquetado/scss/detail.scss [hunk 2]
```diff
@@ -93,3 +95,74 @@
...
+    align-items: center;
+    flex-direction: column;
+    height: 570px;
+    margin-top: 100px;
+    padding: 40px 20px 0 20px;
+    width: 335px;
+  }
+
+  .photo-detail {
+    max-height: 100%;
```
Comment (CrisVega08) [inline maquetado/scss/detail.scss]: max-height and max-width , try to use a exact value for example 

`max-width: 300px;
width: 100%;`

File: maquetado/scss/detail.scss [hunk 3]
```diff
@@ -17,15 +17,17 @@
   width: 260px;
 }
 
+$badge-height: 95px;
+$badge-width: 80px;
```
Comment (CrisVega08) [inline maquetado/scss/detail.scss]: Add empty line and define these in top file

File: maquetado/scss/detail.scss [hunk 4]
```diff
@@ -93,3 +95,74 @@
...
+    max-width: 100%;
+  }
+
+  .photo {
+    height: 300px;
+    margin-bottom: 20px;
+    width: 215px;
+
+    &::after {
+      height: calc(#{$badge-height} - #{$badge-height} * 0.3);
```
Comment (CrisVega08) [inline maquetado/scss/detail.scss:125]: jejeje nice

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: maquetado/scss/detail.scss (hunk 1 above)
Comment (jersonlopez) [reply maquetado/scss/detail.scss]: keep calm bro, this is a class, not a tag

File: maquetado/scss/detail.scss [hunk 5]
```diff
@@ -93,3 +95,74 @@
...
+  .information {
+    width: 100%;
+  }
+
+  .book-description {
+    margin-bottom: 20px;
+  }
+
+  .book-description:first-child {
+    padding-bottom: 10px;
```
Comment (CrisVega08) [inline maquetado/scss/detail.scss]: you can add this style of book-description at line 137 using ampersand character.
```
.class {
  ...
  &:pseudo-class{ }
}
```

File: maquetado/scss/detail.scss (hunk 1 above)
Comment (CrisVega08) [reply maquetado/scss/detail.scss]: jajaja you're right change this name please.

File: maquetado/scss/detail.scss [hunk 6]
```diff
@@ -93,3 +95,74 @@
   font-size: 20px;
   line-height: 23px;
 }
+
+@media only screen and (max-width: 1024px) {
```
Comment (CrisVega08) [inline maquetado/scss/detail.scss]: Use a variable for 1024px; you can create other partial fiel for example _sizes.scss like _colors.scss

File: maquetado/scss/detail.scss [hunk 7]
```diff
@@ -1,4 +1,6 @@
 @import 'colors';
+$badge-height: 95px;
```
Comment (CrisVega08) [inline maquetado/scss/detail.scss]: You can move these definitions to a partial file.

**Review by CrisVega08 — APPROVED** [review]

**Pull request merged**